trueos-handbook/introducing.rst

```diff
@@ -800,17 +800,48 @@ differences between Windows and BSD:
 * General `Comparison of Operating Systems <https://en.wikipedia.org/wiki/Comparison_of_operating_systems>`_
 * `Open Source Alternatives <https://opensource.com/alternatives>`_
 
-.. TODO: Expand this section
-   .. index:: MacOSX and TrueOS
-   .. _compareMacOSX:
 
-   Mac OS X and TrueOS
-   -------------------
+.. index:: MacOSX and TrueOS
+.. _compareMacOSX:
+
+Mac OS X and TrueOS
+-------------------
+
+Mac OS X is related to FreeBSD, resulting in some system level 
+similarities. Many of the features that make Mac OS X a popular 
+operating system are found in |trueos|. |trueos|, like Mac OS X, 
+includes built in securtiy features and access to many free programs 
+and apps. Mac OS X comes with some of these programs pre-installed, 
+but |trueos| users can use the App Cafe to download programs that 
+fill many of the same functions.|true OS|is intentionally a light 
```

Comment:
|true OS| -> |trueos|. Also make sure to have an empty space at the start of the new sentence.